dev/.env.example

```diff
@@ -1,3 +1,25 @@
+# General environment variables
+INSTANCE_NAME=
+TENANT_XLM_BOOTSTRAP_AMOUNT=
+ADMIN_PORT=
+ADMIN_ACCOUNT=
+ADMIN_API_KEY=
+
+# Broker configuration environment variables
+EVENT_BROKER_TYPE=
+BROKER_URLS=
+CONSUMER_GROUP_ID=
+KAFKA_SECURITY_PROTOCOL=
+KAFKA_SASL_USERNAME=
+KAFKA_SASL_PASSWORD=
+KAFKA_SSL_ACCESS_KEY=
+KAFKA_SSL_ACCESS_CERTIFICATE=
+
+# Scheduler environment variables
+ENABLE_SCHEDULER=false                      # default value
+SCHEDULER_PAYMENT_JOB_SECONDS=
+SCHEDULER_RECEIVER_INVITATION_JOB_SECONDS=
```

Comment:
This is not needed, we already set these values on the `docker-compose-sdp-anchor.yml`.